lib/config/migration.ts

```diff
@@ -158,16 +158,19 @@ export function migrateConfig(
     }
     if (
       is.nonEmptyObject(migratedConfig['pip-compile']) &&
-      is.nonEmptyArray(migratedConfig['pip-compile'].fileMatch)
+      is.nonEmptyArray(migratedConfig['pip-compile'].managerFilePatterns)
```

Comment:
This logic assumes that pip-compile.fileMatch would have been migrated to pip-compile.managerFilePatterns _first_ / prior. Is that the case?

Reply:
Yes and there's a test for it as well